src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -187,6 +191,14 @@ class BankAccountStep extends React.Component {
         });
     }
 
+    /**
+    *
+    * @param {Object} value
+    */
+    debouncedUpdateReimbursementAccountDraft(value) {
```

Comment:
Can you add a comment here explaining why this is debounced? An example is [here](https://github.com/Expensify/App/blob/ec9a3190e64a75c61f1feb443236a8a171d067c6/src/pages/home/report/ReportActionCompose.js#L295).

Reply:
I'm not sure I agree that a debounce is the right want to handle this. When should this component re-render when the draft values change? If the answer is "never" and we only need these values when the component is constructed then we should prevent the updates from happening altogether (not debounce them).

Reply:
Using `Why Did You Render`, I couldn't see any unnecessary re-renders because of the props. Maybe I'm not looking at the right place?

I think the intention with debounce in this case is to prevent several unnecessary calls to Onyx.merge, though. Is that something we care about? If not, I'm happy to remove the debounce and use `updateReimbursementAccount` instead.

Reply:
>Is that something we care about?

I don't know. It might be if there's an observable performance downside. I'm asking because **you** are introducing this change and mention this:

>We debounce this method in the constructor so that it's not called too often to update Onyx and re-render this component.

Is this happening or not? Why should we care? Is there some other explanation for why we should use a debounce?

Reply:
Hmmm looking more into this, removing the debounce didn't result in any unnecessary re-renders and I also couldn't notice any performance downgrade. Given that I don't see any other reasons to debounce this function, I will go ahead and remove the debounce.